cmd/convox/apps.go

```diff
@@ -267,6 +267,27 @@ func cmdAppParamsSet(c *cli.Context) error {
 		params[parts[0]] = parts[1]
 	}
 
+	existing, err := rackClient(c).ListParameters(app)
+	if err != nil {
+		return stdcli.Error(err)
+	}
+
+	hasChange := false
+	for k, v := range params {
+		res, ok := existing[k]
+		if !ok {
+			hasChange = true
+			break
+		} else if res != v {
+			hasChange = true
+			break
+		}
+	}
+
+	if !hasChange {
+		return stdcli.Error(fmt.Errorf("No updates are to be performed"))
```

Comment:
Could we direct the user to `convox apps params` to see existing values? Same with the rack params.